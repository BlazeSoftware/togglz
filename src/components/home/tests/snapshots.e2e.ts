import { newE2EPage } from '@stencil/core/testing';

describe('home-page', () => {
  test('renders correctly', async () => {
    const page = await newE2EPage();
    await page.setContent('<home-page></home-page>');
    const element = await page.find('home-page');

    await page.waitForChanges();

    expect(element.outerHTML).toMatchSnapshot();
  });
});
