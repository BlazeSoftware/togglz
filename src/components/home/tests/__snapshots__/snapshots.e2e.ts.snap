// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home-page renders correctly 1`] = `"<home-page class=\\"hydrated\\"><div class=\\"o-home-page u-centered u-absolute-center u-large\\"><h1 class=\\"c-heading u-super\\">Togglz</h1><blaze-toggle class=\\"u-super hydrated\\" type=\\"brand\\"><label class=\\"c-toggle c-toggle--brand\\"><input type=\\"checkbox\\" aria-checked=\\"true\\"><div class=\\"c-toggle__track\\"><div class=\\"c-toggle__handle\\"></div></div></label></blaze-toggle><h2 class=\\"c-heading u-medium u-text--quiet c-tagline\\">Feature toggle service for websites and apps</h2></div></home-page>"`;
